src/tools/dynamic/sf-list-all-tools.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SfMcpServer } from '../../sf-mcp-server.js';
+import { textResponse } from '../../shared/utils.js';
+import { listAllTools } from '../../shared/tools.js';
+
+export function registerToolListAllTools(server: SfMcpServer): void {
+  server.tool(
+    'sf-list-all-tools',
```

Comment:
To follow the naming pattern (sf-verb-noun) should we name this `sf-list-tools` and imply the "all" part?  imo, the same applies to `sf-list-all-orgs`.  Or are we expecting to have tools that list something other than all?